src/server/utilities/cspHeader/index.js

```diff
@@ -18,7 +18,7 @@ const directives = {
       'https://pagead2.googlesyndication.com', // ads
       'https://tpc.googlesyndication.com', // ads
       'https://a1.api.bbc.co.uk/hit.xiti', // ATI
-      'https://toggles.api.bbci.co.uk', // Toggles service
+      'https://config.api.bbci.co.uk/', // Toggles service
```

Comment:
For the CSP headers, you should just have the domain and not the trailing `/`.

```suggestion
      'https://config.api.bbci.co.uk', // Toggles service
```